robot_calibration/include/robot_calibration/capture/ground_plane_finder.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2014-2015 Fetch Robotics Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ROBOT_CALIBRATION_CAPTURE_GROUND_PLANE_FINDER_H
+#define ROBOT_CALIBRATION_CAPTURE_GROUND_PLANE_FINDER_H
+
+#include <ros/ros.h>
+#include <pcl/point_types.h>
+#include <pcl_ros/point_cloud.h>
```

Comment:
we can't include these two files (this line, and above it) as we aren't depending on PCL anymore